Apps/Database/src/Delegates/DBCommentDelegate.cs

```diff
@@ -59,6 +59,19 @@ public DBResult<IEnumerable<Comment>> GetList(string hdId, string parentEntryId)
             return result;
         }
 
+        /// <inheritdoc />
+        public DBResult<IEnumerable<Comment>> GetAll(string hdId)
```

Comment:
Unit tests for this method may need to be added if possible?

Reply:
same as above, merge from dev